Governance/manage/manage-file.md

```diff
@@ -1,17 +1,45 @@
-# File organisation
+# File handling
+
+This part covers how to manage the actual files and directories, starting from the download process. It applies mostly to replicated datasets, as how to name and organise the files of a new dataset is covered already in the Creating dataset guidelines. 
 
 ## Managing downloads
 
-Bash script are ok if the downloads are quite simple, if there is a lot of shuffling of files and directories to do, soon bash become unruly and you'll find you have to use obscure directive to reproduce data structure which are default for other languages
-We use a lot of python codes as soon as a diownload code becomes unwielding with bash
+How to manage downloads depends mostly on how the original data is distributed. Often servers offer more than one option to download the data. It will depend also on how big the dataset is, in some cases it's possible that a dataset is composed of one or few files that could easily be downloaded from a web interface. 
+Even in such simple cases and where regular updates are not needed is good practice to create a code. A code is a way to document exactly where the data was downloaded from and how. 
+Bash scripts are fine if the downloads are quite simple, as in the above example of few files that we can retrieve with a simple `wget` or `curl`. If there is a lot of shuffling of files and directories to do, soon a bash script would become complicated and unreadable, in these cases you should use a proper coding language. 
```

Comment:
, as in the above example of few files that we can retrieve with a simple `wget` or `curl` --> ; e.g., in the above example, we can retrieve data with a simple `wget` or `curl`

Reply:
I read this information about bash and python script in the previous commit, too, not sure if it's duplicated.

Reply:
I don't think this suggested edit (as in the above -> e.g.) makes sense as it effectively adds 'example' repetition. I think the original new text is fine. Unsure about duplication though I haven't read through everything yet but I'll keep an eye out for that.

Reply:
I'm quite sure it's not repeated I run a grep on all the files and appears only once